x-pack/metricbeat/modules.d/prometheus.yml.disabled

```diff
@@ -23,6 +23,16 @@
   # Store counter rates instead of original cumulative counters (experimental, default: false)
   #rate_counters: true
 
+# Metrics sent by a Prometheus server using remote_write option
+#- module: prometheus
+#  metricsets: ["remote_write"]
+#  host: "localhost"
+#  port: "9201"
+
+  # Secure settings for the server using TLS/SSL:
+  #ssl.certificate: "/etc/pki/server/cert.pem"
```

Comment:
Nit: Should these two lines be consistent with the rest? Something like this:
```suggestion
#  ssl.certificate: "/etc/pki/server/cert.pem"
```

Reply:
Just wanted to avoid giving the option to "uncomment all" since these are extra options :)